rust/src/polars/mod.rs

```diff
@@ -57,6 +57,10 @@ where
                 return Ok(None);
             }
 
+            if !kwargs.is_empty() {
+                return fallible!(FailedFunction, "OpenDP does not allow pickled keyword arguments as they may enable remote code execution.");
```

Comment:
My understanding is that pickle is specific to python, but polars also supports R: If any eventual R code might also exercise this path, should the error message be more language-agnostic?

Reply:
Polars doesn't directly support plugins from R at this point, but if you know how things work, you can still write those DSL nodes for plugins into expressions in R. Now that we don't use pickling, R will be able to fill kwargs fields in these new DSL nodes with empty strings.


Ultimately, the slot to fill in these DSL nodes is binary pickle data, so if someone were to craft a DSL node with data in there from R, the error is still appropriate.